homeassistant/components/idasen_desk/button.py

```diff
@@ -83,8 +86,9 @@ def __init__(
         """Initialize the IdasenDesk button entity."""
         self.entity_description = description
 
-        self._attr_name = f"{device_info[ATTR_NAME]} {self.entity_description.key}"
+        self._attr_name = f"{self.entity_description.name}"
         self._attr_unique_id = f"{self.entity_description.key}-{address}"
+        self._attr_icon = self.entity_description.icon
```

Comment:
```suggestion
```

Reply:
If unset, uses this already https://github.com/home-assistant/core/blob/5d430f53cd87143f724bc0a4e8f960e4d6c51252/homeassistant/helpers/entity.py#L565